contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -19,19 +22,36 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     string name;
   }
 
-  struct Node {
+  struct NodeParams {
     /// @notice The id of the node operator that manages this node
-    uint256 nodeOperatorId;
+    uint32 nodeOperatorId;
+    /// @notice The signer address for application-layer message verification.
+    address signer;
```

Comment:
Is this the onchain signing key? `address` only works for EVM - `bytes` is more general (maybe `bytes32` also works, I don't think there are any chains that have public keys > 32 bytes atm)

Reply:
If this is the offchain signing public key, its too small (Ed25519 used there)

Reply:
I've created a follow up ticket to address this